spec/rails_helper.rb

```diff
@@ -85,20 +120,31 @@
     raise StandardError.new "\"#{example.full_description}\" in #{example.location} timed out."
   end
 
-  config.around :each, :disable_bullet do |example|
-    Bullet.raise = false
-    example.run
-    Bullet.raise = true
-  end
-
-  def pre_transition_aged_youth_age
-    Date.current - CasaCase::TRANSITION_AGE.years
-  end
```

Comment:
Weird place for a helper, and only used a handful of times in two files, I just replaced them in the files.